src/app/page.tsx

```diff
@@ -66,13 +67,42 @@ const Home = async ({ searchParams }: Props) => {
                 </p>
               </div>
             </div>
-            {remainingDrops?.length > 0 && (
+            {drops?.length > 4 && (
```

Comment:
Might be better to use a ternary here

i.e. 
```suggestion
            {drops?.length > 4 ? ( ...
```

Reply:
Much cleaner, good suggestion. Done ✅